.tx/config

```diff
@@ -20,14 +20,9 @@ file_filter = MapboxCoreNavigation/Resources/<lang>.lproj/Localizable.strings
 source_file = MapboxCoreNavigation/Resources/Base.lproj/Localizable.strings
 source_lang = en
 
-[mapbox-navigation-ios.swiftexample]
-file_filter = Examples/Swift/<lang>.lproj/Main.strings
-source_file = Examples/Swift/en.lproj/Main.strings
-source_lang = en
-
-[mapbox-navigation-ios.objcexample]
-file_filter = Examples/Objective-C/<lang>.lproj/Main.strings
-source_file = Examples/Objective-C/en.lproj/Main.strings
+[mapbox-navigation-ios.ExampleMainStrings]
+file_filter = Examples/<lang>.lproj/Main.strings
+source_file = Examples/en.lproj/Main.strings
```

Comment:
Whoops, I left the `s` in “Examples”. Fixed in a207536bee604b42dbc09cbdb0bc2dda5bf67233.